ui/src/state/lure/lure.ts

```diff
@@ -136,14 +136,14 @@ export const useLureState = create<LureState>(
           undefined
         );
 
-        const enableAcked = await asyncWithDefault(
+        const enableAcked = !(await asyncWithDefault(
           () =>
             api.scry<boolean>({
               app: 'reel',
               path: `/outstanding-poke/${flag}`,
             }),
           true
```

Comment:
does the default need to change too or is it good?

Reply:
The default is good